modules/nw-multus-network-json.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * networking/multiple_newtorks/networking-gitops.adoc
+
+[id="nw-multus-network-json_{context}"]
+= Configuration for network attachment definition
+
+The following sections provide reference information about the network plugins that you can use when you create a JSON-formatted network attachment definition.
+
+== Configuration for bridge
+
+include::nw-multus-bridge-object.adoc[tag=json-ref]
```

Comment:
Would need to confirm that we're actually allowed to do this, both the include and the tag style.

Reply:
I suppose that I could "single source" it--duplicate the source every single time.  Seriously, I have no strong feeling and this PR is really to trigger conversation.

Reply:
@mikemckiernan, oh, I thought in that case you could just put the includes in the assembly instead?